loggingpy/log.py

```diff
@@ -197,26 +201,28 @@ def log(self, log_entry: LogEntry):
 
     def write_entry(self, log_level: Enum, payload_type: str, message: str='', data=None, exception: Exception = None):
 
-        if self.prefix_payload_type and self.context is not None and self.context is not "":
-            payload_type = self.context + '.' + payload_type
+        if self.context and self.prefix_payload_type:
+            if payload_type:
```

Comment:
What happens now if the payload type is not supplied?

Reply:
Both of this is fixed in the newest version of the code. It seems I pushed it right before you completed your comment.

Reply:
Strangely it said it invited you only after I pushed that one. Are you on your warp-speed vehicle so that time is relativistic and we can no longer evaluate the order of two events?

Reply:
🚀